scripts/jira_tickets/jira_recurrent_tickets.vertebrates.json

```
@@ -221,6 +221,42 @@
             "summary": "Import the Alt-allele groups",
             "name_on_graph": "Alt-alleles import"
          },
+         {
+             "component": "Production tasks",
+             "description": "https://github.com/Ensembl/ensembl-compara/blob/release/97/docs/production/READMEs/pair_aligner_patches.rst",
+             "summary": "Merge LastZ_Patch human alignments",
+             "name_on_graph": "LastZ_Patch:Human"
+         },
+         {
+             "component": "Production tasks",
+             "description": "https://github.com/Ensembl/ensembl-compara/blob/release/97/docs/production/READMEs/pair_aligner_patches.rst",
+             "summary": "Merge LastZ_Patch mouse alignments",
+             "name_on_graph": "LastZ_Patch:Mouse"
+         },
+         {
+             "component": "Production tasks",
+             "description": "https://github.com/Ensembl/ensembl-compara/blob/release/97/docs/production/READMEs/pair_aligner_patches.rst",
+             "summary": "Merge LastZ_Patch zebrafish alignments",
+             "name_on_graph": "LastZ_Patch:Zebrafish"
+         },
+         {
+             "component": "Production tasks",
+             "description": "https://www.ebi.ac.uk/seqdb/confluence/display/EnsCom/Merge+the+DNA+data#MergetheDNAdata-Haplotypesandassemblypatchesagainstotherspecies",
```

I would link to the entire section about patch alignments:
```suggestion
             "description": "https://www.ebi.ac.uk/seqdb/confluence/display/EnsCom/Merge+the+DNA+data#MergetheDNAdata-Patchalignments",
```